packages/experimental/src/smart-session/utils/ConstantUtils.ts

```diff
@@ -0,0 +1,3 @@
+export const ConstantsUtil = {
+  WC_COSIGNER_BASE_URL: 'https://rpc.walletconnect.org/v1/sessions'
```

Comment:
We should change the variable name to not contain WC

Reply:
changed `WC_COSIGNER_BASE_URL` to `COSIGNER_BASE_URL`